.circleci/config.yml

```diff
@@ -1688,7 +1691,7 @@ jobs:
       - run: git config --global core.autocrlf false
       - checkout
       # Ensure windows has python3 alias required by prepare_bytecode_report
-      - run: ln -s /c/tools/miniconda3/python /c/tools/miniconda3/python3
+      - run: ln -s /c/ProgramData/chocolatey/bin/python3.12 /c/ProgramData/chocolatey/bin/python3
```

Comment:
The changes looks good to me, but it would be helpful to understand why `miniconda3` no longer exists in `/c/tools`. Its sudden absence is unexpected, especially because we rely on a specific version of the [Windows orb](https://github.com/ethereum/solidity/blob/9d988ccefd62bd77f2848767749d240fb3981527/.circleci/config.yml#L36), even though it uses the [default Windows image within the orb](https://github.com/ethereum/solidity/blob/9d988ccefd62bd77f2848767749d240fb3981527/.circleci/config.yml#L609).

After thinking a bit about it, my concern with the current approach is the potential for future breaks if the Python version is updated. Perhaps we could reinstall `miniconda3`, using the command: `choco install miniconda3 -y`, and then create the symlink as previously done.

Reply:
Yeah, I would tend to agree - or just symlink the regular (non choco) python to python3; at least if that one gets upgrade, the symlink should still work (unless it get's updated to python 4 at some point :D).

In which step should the miniconda install be performed? Same as the symlink?

Reply:
> Yeah, I would tend to agree - or just symlink the regular (non choco) python to python3; at least if that one gets upgrade, the symlink should still work (unless it get's updated to python 4 at some point :D).

haha yeah
 
> In which step should the miniconda install be performed? Same as the symlink?

Yes, I think it could be the same job (`b_bytecode_win`) one step before the symlink.

Reply:
We don't need `miniconda` for anything. The only reason it's even mentioned by name is that we needed `python3` to point at something to satisfy scripts (930c8c075344640dfe16e3f1131b6462c32c9990). If there's another Python 3 interpreter in the image, it's fine to use it.

I'd rather not have it point specifically at something with a version embedded. We'll have to update that constantly. Does Windows allow a symlink that just points at the global `python` (without a path specified)? Or some kind of alias? Or even a Bash function?

Reply:
I haven't checked, but I don't see why not. And yes, there's a default Python interpreter as well as the choco one, and they're both the same version. I'll update it to point to the default one.

Reply:
> We don't need `miniconda` for anything. The only reason it's even mentioned by name is that we needed `python3` to point at something to satisfy scripts ([930c8c0](https://github.com/ethereum/solidity/commit/930c8c075344640dfe16e3f1131b6462c32c9990)). If there's another Python 3 interpreter in the image, it's fine to use it.
> 
> I'd rather not have it point specifically at something with a version embedded. We'll have to update that constantly. Does Windows allow a symlink that just points at the global `python` (without a path specified)? Or some kind of alias? Or even a Bash function?

I pushed the change as a separate commit, but the other one is installed in `/c/Python312`, which is 'versioned' as well, although less so than the choco binary, which would likely be removed in case of an upgrade.

Reply:
What about other options? E.g. creating a script?

Anyway, if this does not work, I'd just replace `python3` with `python` in those few commands that fail (I think it's just the two `python3` calls in `parallel_bytecode_report.sh`). We don't use `python3` in other Windows jobs, like `t_win_pyscripts`, either.  These days it's pretty rare to find Python 2 so it's not like it matters that much and even back then it was just a way to get the requirement more explicit, but IMO just `python` is perfectly fine, even in environments that do have the `python3` alias.